rust-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "1.79.0"
```

Comment:
is this intended?

if so, could have been separate PR as it's unrelated

Reply:
Ah I had to add it to get it to work locally. Will remove.